src/include/conv_fin.hpp

```diff
@@ -724,15 +724,6 @@ int ConvFin<Tgpu, Tref>::MIOpenPerfEval()
                 res_item["bias"]           = ctx.problem.bias;
                 res_item["kernel_objects"] = kern_objs;
                 res_item["reason"]         = "Success";
-
-                if(s.IsTunable())
```

Comment:
Just curious. We do not need this test any more?

Reply:
If you look into MIOpen/src/include/miopen/any_solver.hpp: you can see that GetPerfCfgParams calls the same validation functions as is called in TestPerfCfgParams, so no new information is generated here.

Reply:
got it!